docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
```

Comment:
I'm a bit confused here... this example specifically shows moving a from and to quickstart app and that is said clearly on the second paragraph:
> Here we use a project made with the [official QuickStart](https://github.com/angular/quickstart) as an example, but you should be able to adjust these instructions to other setups.

What makes you say this example isn't showing the quickstart details?